cmd/dt/login.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"io"
+	"os"
+	"strings"
+
+	"github.com/docker/cli/cli/config"
+	"github.com/docker/cli/cli/config/types"
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/spf13/cobra"
+	"github.com/vmware-labs/distribution-tooling-for-helm/internal/log"
+)
+
+var loginCmd = newLoginCmd()
+
+type loginOptions struct {
+	serverAddress string
+	user          string
+	password      string
+	passwordStdin bool
+}
+
+func newLoginCmd() *cobra.Command {
+	var opts loginOptions
+
+	cmd := &cobra.Command{
```

Comment:
Similar to above, please look at the existing commands as all need to be consistent. Needs a long description, needs an example. Also I'm thinking that it is probably better to use `REGISTRY` rather than `SERVER`, as that is what you are referring to in the cmd description.